plugins/channel_header_plug/channel_header_plug.tsx

```diff
@@ -250,6 +256,36 @@ class ChannelHeaderPlug extends React.PureComponent<ChannelHeaderPlugProps, Chan
         );
     }
 
+    handleOpenMarketplace = () => {
+        this.props.actions.openModal({
+            modalId: ModalIdentifiers.PLUGIN_MARKETPLACE,
+            dialogType: MarketplaceModal,
+            dialogProps: {openedFrom: 'channel_header'},
+        });
+    };
+
+    createMarketplaceButton = () => {
+        const {formatMessage} = this.props.intl;
+
+        return (
+            <HeaderIconWrapper
+                key='channelHeaderButton-marketplace'
+                buttonClass='channel-header__icon'
+                iconComponent={(
+                    <Icon
+                        size={16}
+                        glyph={'view-grid-plus-outline'}
+                    />
+                )}
+                onClick={this.handleOpenMarketplace}
+                buttonId='channel_header_plug_marketplace'
+                tooltipKey='plugin'
+                tooltipText={formatMessage({id: 'channel_header_plug.marketplace', defaultMessage: 'App Marketplace'})}
```

Comment:
Why does the tooltip say `App Marketplace`? The Marketplace currently only contains a single app and the rest are plugins.

Reply:
@hanzei  As specified in the ticket, this should follow the current implementation in the post hover menu: Message actions > Apps Marketplace
<img width="293" alt="Screen Shot 2023-03-06 at 11 44 13 AM" src="https://user-images.githubusercontent.com/79058848/223176059-eb1c7770-407c-46a3-a094-b6b619969de1.png">

Should texts be changed in both instances to just 'Marketplace'?
BTW it reads this way in the product menu:
<img width="284" alt="Screen Shot 2023-03-06 at 11 49 40 AM" src="https://user-images.githubusercontent.com/79058848/223176775-ffa3d0d0-0d5a-4091-825d-5d9609c8cf0d.png">

Last thing, noticed that we'll have a difference in the icon used in the marketplace item. Should we also make all the instances to match? 
- <img width="30" alt="Screen Shot 2023-03-06 at 11 52 58 AM" src="https://user-images.githubusercontent.com/79058848/223177738-a4528f2d-d184-44bf-93e7-2f6073d4e9c4.png"> `view-grid-plus-outline`
- <img width="30" alt="Screen Shot 2023-03-06 at 11 53 12 AM" src="https://user-images.githubusercontent.com/79058848/223177806-fd1e85e3-3921-449d-b85a-d435314ddab8.png"> `apps`

cc @esethna @marianunez @annelieseklein

Reply:
Here's my proposal:

- Opening the marketplace should use the `view-grid-plus-outline`, since users are opening the marketplace to add new apps/plugins. Propose we change that icon in the header product menu so we have consistency with the post hover menu and apps bar icon. 
- We should call it the "App Marketplace". Even though it's technically mostly plugins, the terminology of "Apps" helps users understand what the marketplace is for. 

cc// @asatkinson @julmondragon